managed/resource_cluster.go

```diff
@@ -270,6 +270,37 @@ and modify the backup schedule of the cluster being created.`,
 							},
 						}),
 					},
+					"azure_cmk_spec": {
+						Description: "AZURE CMK Provider Configuration.",
+						Optional:    true,
+						Attributes: tfsdk.SingleNestedAttributes(map[string]tfsdk.Attribute{
+							"client_id": {
+								Description: "Client ID",
```

Comment:
A bit more description for the field will be nice, knowing this will be the final docs.

Reply:
Done